### ISSUE TITLE
Pixel Font Aliasing

### ISSUE THREAD

**Issue by FoxMoss** [OLIVER427-c6ret/oliver427game#1]
Consider using a better font rendering setup since aliasing looks funny with pixel fonts. Been a while since I used Unity but I believe TextMeshPro fixes most issues. 

![Screenshot from 2024-01-31 18-08-21](https://github.com/OLIVER427-c6ret/oliver427game/assets/46651354/6fb05174-fd4c-44e9-9123-576bb081acbb)

**Comment by FoxMoss** [comment]
Also the main menu doesn't seem to be using Unity's canvas system properly, but its a bit to learn.

![image](https://github.com/OLIVER427-c6ret/oliver427game/assets/46651354/577d4760-2f68-4697-b719-0b0a746e17b1)

**Comment by OLIVER427-c6ret** [comment]
yeah I made that game about 2 years ago. it sucks, but I might consider improving it (and possibly using my OWN assets) sometime
